core/src/test/java/kafka/server/share/SharePartitionManagerTest.java

```diff
@@ -2368,7 +2368,7 @@ public SharePartitionManager build() {
                     persister,
                     delayedShareFetchPurgatory,
                     delayedActionsQueue,
-                    new GroupConfigManager(new HashMap<>()),
+                    mock(GroupConfigManager.class),
```

Comment:
Hmmm, though it's fine to mock it but why we did if we are not changing any handling in SharePartitionManagerTests?

Reply:
Hi, thanks for the review. This change should have gone in the previous PR. I got a comment to change the instance of GroupConfigManager to a mock, but I missed that change in SharePartitionManager. I can create a separate PR for this as well if you like.

Reply:
It's fine to keep in this PR, thanks for the context.